src/lib/prismaUtils.ts

```diff
@@ -63,41 +61,23 @@ export function findAdvancedDelegatee({
 
   switch (namespace) {
     case TENANT_NAMESPACES.OPTIMISM:
-      return prisma.optimismAdvancedDelegatees.findMany(
-        partial ? undefined : condition
-      );
+      return prisma.optimismAdvancedDelegatees.findMany(condition);
     case TENANT_NAMESPACES.ENS:
-      return prisma.ensAdvancedDelegatees.findMany(
-        partial ? undefined : condition
-      );
+      return prisma.ensAdvancedDelegatees.findMany(condition);
     case TENANT_NAMESPACES.ETHERFI:
-      return prisma.etherfiAdvancedDelegatees.findMany(
-        partial ? undefined : condition
-      );
+      return prisma.etherfiAdvancedDelegatees.findMany(condition);
     case TENANT_NAMESPACES.UNISWAP:
-      return prisma.uniswapAdvancedDelegatees.findMany(
-        partial ? undefined : condition
-      );
+      return prisma.uniswapAdvancedDelegatees.findMany(condition);
     case TENANT_NAMESPACES.CYBER:
-      return prisma.cyberAdvancedDelegatees.findMany(
-        partial ? undefined : condition
-      );
+      return prisma.cyberAdvancedDelegatees.findMany(condition);
     case TENANT_NAMESPACES.SCROLL:
```

Comment:
I must have missed the time when we introduced this pattern. Why are we doing this?

Reply:
It's because the Prisma upgrade, Prisma got more strict with their API.  No choice.  Plan is to kill off, using RAW SQL and/or DAONode.